postgresql-persistence/src/main/java/com/netflix/conductor/aurora/AuroraExecutionDAO.java

```diff
@@ -511,8 +532,23 @@ public void resetStartTime(Task task, boolean updateOutput) {
 	}
 
 	private Task getTask(Connection tx, String taskId) {
-		String GET_TASK = "SELECT json_data FROM task WHERE task_id = ?";
-		return query(tx, GET_TASK, q -> q.addParameter(taskId).executeAndFetchFirst(Task.class));
+		String GET_TASK = "SELECT json_data, start_time, end_time FROM task WHERE task_id = ?";
+		return query(tx, GET_TASK, q ->{
+			ResultSet rs = q.addParameter(taskId).executeQuery();
+			if ( rs.next()){
+				String json = rs.getString("json_data");
+				try {
+					Task task = new ObjectMapper().readValue(json, Task.class);
```

Comment:
The `om` (instead of new ObjectMapper()) instance already in the class. It shall be available for the function to be used